docs/explanation/intro-to-ontologies.md

```diff
@@ -0,0 +1,233 @@
+# Introduction to ontologies
+
+Based on [CL editors training](https://docs.google.com/presentation/d/11WeCHCeGYSPEO7hUYFTdPivptxX4ajj5pVHDm24j4JA) by David Osumi-Sutherland
+
+## Why do we need ontologies 
+
+### We can't find what we're looking for
+- Too flexible, multiple options, no enforcement standards 
+    - standardized
+    - quasi-standardized
+    - non-standardized
+- Data at varying depth or granularity
+
+For example, trying to refer to feces, in NCBI BioSample:
+
+| Query            | Records    |
+| ---------------- | ---------- |
+| Feces            | 22,592     |
+| Faeces           | 1,750      |
+| Ordure           | 2          |
+| Dung             | 19         |
+| Manure           | 154        |
+| Excreta          | 153        |
+| Stool            | 22,756     |
+| Stool NOT faeces | 21,798     |
+| Stool NOT feces  | 18,314     | 
+
+### We don't know what we're talking about
+
+Because we have no single reference glossary the result is millions of statements that are not obviously related
+ 
+Here is the male genitalia of a gasteruptiid wasp, and these 5 different structures here have each been labeled "paramere" - by different people, studying different hymenopteran lineages - how do we know what "paramere" means when it is referred to? 
```

Comment:
Minor structural edits:

Here is the male genitalia of a gasteruptiid wasp, and these 5 different structures here have each been labeled "paramere" by different people, each studying different hymenopteran lineages. How do we know what "paramere" means when it is referred to?